example/webpack.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
     }),
     new HtmlWebpackPlugin(),
     new AddAssetHtmlPlugin({
-      filepath: require.resolve('./build/vendor.dll.js'),
+      filepath: path.resolve(__dirname, './build/*.dll.js'),
```

Comment:
so sorry, I just make a mistake. This is necessary for the `npm run example` command to find  files.